Assets/HandTracking/Scripts/ToggleMRTKHandVisualization.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit;
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.OpenXR.Samples
+{
+    /// <summary>
+    /// Turns the MRTK hand visualization off when the scene is loaded, to defer to the sample scene's representation.
```

Comment:
Love this comment.  This sample itself is already a very good example of how to use MRTK.  I will remember this.

Can you help add more comments to other sample code as well when you get there?  Thanks